code/modules/antagonists/wizard/abilities/animatedead.dm

```diff
@@ -20,13 +20,13 @@
 			return
 		if(!isdead(target))
 			boutput(holder.owner, "<span class='alert'>That person is still alive! Find a corpse.</span>")
-			return 1 // No cooldown when it fails.
+			return TRUE // No cooldown when it fails.
 		if(!istype(get_area(holder.owner), /area/sim/gunsim))
 			holder.owner.say("EI NECRIS", FALSE, maptext_style, maptext_colors)
 		..()
 
-		var/obj/critter/magiczombie/UMMACTUALLYITSASKELETONNOWFUCKZOMBIESFOREVER = new /obj/critter/magiczombie(get_turf(target)) // what the fuck
-		UMMACTUALLYITSASKELETONNOWFUCKZOMBIESFOREVER.CustomizeMagZom(target.real_name, ismonkey(target))
```

Comment:
This is possibly the worst var name I've seen in this codebase and that's saying something